### PR TITLE
resolve #48 using slash as fragment separator

### LINKED ISSUE #48: slash vs. hash
I think the particular aspect of slash vs. hash recently discussed in https://github.com/ESIPFed/sweet/issues/46 deserves its own entry for further discussion and resolution.

### PR DESCRIPTION
**Pull request by carueda**
Here's the PR ready for when folks think a decision for using slash has been made.

Change done via a regex-based replacement applied on all the *.ttl files:

`http://sweetontology.net/(.+)#` -> `http://sweetontology.net/$1/`

### CODE REVIEW

**Comment by graybeal** [comment]
from what I know of ontologies, this should work. I don't think the separator is ever referenced without the base path also being present.

**Comment by lewismc** [comment]
@carueda I looked through (sanity checked) the remainder of the proposed changes... the issue I highlighted DOES NOT occur anywhere else other than the instances I've highlighted.

File: humanAgriculture.ttl [hunk 1]
```diff
@@ -14,80 +14,80 @@
...
 _:node1boamdscqx4 a owl:Restriction ;
-	owl:onProperty <http://sweetontology.net/relaSci#hasSpecies> ;
-	owl:allValuesFrom <http://sweetontology.net/matrAnimal#Silkworm> .
+	owl:onProperty <http://sweetontology.net/relaSci/hasSpecies> ;
+	owl:allValuesFrom <http://sweetontology.net/matrAnimal/Silkworm> .
 
-<http://sweetontology.net/humanAgriculture#ShellfishProduction> a owl:Class ;
-	rdfs:subClassOf <http://sweetontology.net/humanAgriculture#AnimalProduction> , <http://sweetontology.net/humanAgriculture#Aquaculture> .
+<http://sweetontology.net/humanAgriculture/ShellfishProduction> a owl:Class ;
+	rdfs:subClassOf <http://sweetontology.net/humanAgriculture/Aquaculture> .
```
[the remark this answers is not in the document]
Comment (carueda) [reply humanAgriculture.ttl]: Thanks for revieweing .... Yikes, my regex is wrong!  sorry, I'll fix all of this.

**Comment by carueda** [comment]
Thanks again @lewismc for spotting my silly mistake.
I re-did the replacement and all files with a strict and appropriate regex `http://sweetontology.net/([a-zA-Z0-9]+)#`.  Then verified that no matches for `http://sweetontology.net/(.+)#` remained.

**Comment by carueda** [comment]
Hi @lewismc , only you have provided an answer here (perhaps others are not aware of the approval mechanism in github)... just a heads-up, perhaps you are going to go ahead and merge. Thx.

**Comment by graybeal** [comment]
+1
There is that what I'm supposed to do? Sorta figured y'all had it sorted.

**Comment by carueda** [comment]
https://help.github.com/articles/approving-a-pull-request-with-required-reviews/

**Comment by lewismc** [comment]
@carueda are you able to merge this into master?

**Comment by carueda** [comment]
@lewismc Yes, I have that permission ... although I was initially expecting you will do the merge itself. No problem my doing it but please confirm that's what you want. Thx.

Btw, with @graybeal 's recent approval, "slash" has officially won ; )

**Pull request merged**